src/mars.c

```diff
@@ -1193,6 +1194,7 @@ Language changes listed by -transition=id:\n\
         VersionCondition::addPredefinedGlobalIdent("D_NoBoundsChecks");
 
     VersionCondition::addPredefinedGlobalIdent("D_HardFloat");
+    objc_tryMain_dObjc();
```

Comment:
For me it's not a question of whether or not GCC supports, it's a question of why you are poisoning my glue interface with names like this?